services/storage/tests/unit/test_dsm_soft_links.py

```diff
@@ -124,7 +124,7 @@ async def test_create_soft_link(
     # assert output_file.last_modified < link_file.fmd.last_modified
 
     # can find
-    files_list = await simcore_s3_dsm.search_read_access_files(
+    files_list = await simcore_s3_dsm.search_owned_files(
```

Comment:
question: I guess there is no test of the handler itself. and that is a bit sad. 
There are tests for almost all the entry points of storage regarding files in ```test_handlers_files.py```. Using this and adding this search entry point could be beneficial also to check that it does not work with anything else than ```kind=owned```

Reply:
Yep, very good point. I will add a test. I believe a `pydantic.ValidationError` should be raised if the kind is not set correctly. But I will add a test to be sure.